src/templates/edit.tsx

```diff
@@ -58,6 +58,24 @@ const Edit: () => JSX.Element = () => {
   const [saveState, setSaveState] = useState<SaveState>();
   const [saveStateFetched, setSaveStateFetched] = useState<boolean>(false); // needed because saveState can be empty
   const [devPageSets, setDevPageSets] = useState<any>(undefined);
+  const [hasParent, setHasParent] = useState<boolean>(true);
+
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      const ancestors = window.location.ancestorOrigins;
+      if (ancestors.length === 0) {
+        setHasParent(false);
+      } else if (!ancestors[0].includes("pagescdn") && !ancestors[0].includes("yext.com")) {
```

Comment:
Does this handle all envs (dev, qa, sbx, prod, euqa, euprod)?

Reply:
tested in dev and prod, will create sites and test for the rest

Reply:
done